openstates/fl/legislators.py

```diff
@@ -132,9 +132,9 @@ def scrape_reps(self, term):
             link = div.xpath('.//div[@class="rep_style"]/a')[0]
             name = link.text_content().strip()
 
-            if 'Vacant' in name or \
-                    'Resigned' in name or \
-                    'Pending' in name:
+            term_details = div.xpath(
+                './/div[@class="term_style"]/a')[0].text_content()
```

Comment:
The `term_style` divs contain no anchors, but once you remove the trailing `/a` in the xpath, it works.